styles.css

```diff
@@ -2,57 +2,65 @@
 	box-sizing:border-box;
 }
 
+body{
+	min-width:550px;
```

Comment:
smells like code smell!

Reply:
This was put in to limit the viewport size...as I didn't want to find solutions for a smaller screen size.